src/js/view/components/menu.js

```diff
@@ -2,40 +2,50 @@ define([
     'view/components/tooltip',
     'utils/helpers',
     'utils/underscore',
+    'events/events',
+    'utils/ui',
     'handlebars-loader!templates/menu.html'
-], function(Tooltip, utils, _, menuTemplate) {
-
+], function(Tooltip, utils, _, events, UI, menuTemplate) {
     var Menu = Tooltip.extend({
         setup : function (list, selectedIndex) {
-            if(this.content){
-                this.content.removeEventListener('click', this.selectListener);
-                this.removeContent();
+            if(!this.iconUI){
+                this.iconUI = new UI(this.el).on('tap', utils.noop);
```

Comment:
Unnecessary now.  There was an issue previously with how 'off' was working when everything was defined in the constructor.  Not an issue now so it's getting removed.